addons/web_editor/tools.py

```diff
@@ -138,27 +138,43 @@ def get_video_embed_code(video_url):
     return Markup('<iframe class="embed-responsive-item" src="%s" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowFullScreen="true" frameborder="0"></iframe>') % data['embed_url']
 
 
-def get_video_thumbnail(video_url):
-    """ Computes the valid thumbnail image from given URL
+def get_video_thumbnail_url(video_url):
+    """ Computes the valid thumbnail image URL from given URL
         (or None in case of invalid URL).
     """
     source = get_video_source_data(video_url)
     if source is None:
         return None
 
-    response = None
+    thumbnail_url = None
     platform, video_id = source[:2]
     if platform == 'youtube':
-        response = requests.get(f'https://img.youtube.com/vi/{video_id}/0.jpg', timeout=10)
+        thumbnail_url = f'https://img.youtube.com/vi/{video_id}/0.jpg'
     elif platform == 'vimeo':
-        res = requests.get(f'http://vimeo.com/api/oembed.json?url={video_url}', timeout=10)
+        video_details = f'http://vimeo.com/api/oembed.json?url={video_url}'
+        res = requests.get(video_details, timeout=10)
```

Comment:
Nitpicking: maybe this is an useless change :shrug:

Reply:
It was to make it clear that the vimeo url leads to an object containing the video details and not directly to the thumbnail, as I didn't find it straightforward to understand :slightly_smiling_face: 
Should I still remove this change ?

Reply:
If you think this helps, then you can keep it. Although, name it `video_details_url` or `url_to_video_details` then (this is what this variable contains: an URL) :+1: